Rakefile

```diff
@@ -444,13 +457,14 @@ which are simply available in the repository.
     port = args.to_hash.fetch(:port, 5000)
     require 'mkmf'
     autobuild_cmd = find_executable 'sphinx-autobuild'
```

Comment:
where does sphinx-autobuild come from?  I see sphinx-autogen in my venv created with the requirements.txt, but not sphinx-autobuild

Reply:
looks like we need a sphinx-autobuild entry in requirements.txt.  Would you please add with the appropriate version ranges you have gotten to work.

Reply:
@lnemsick-simp :ballot_box_with_check: